src/components/mixins/message.js

```diff
@@ -13,6 +13,18 @@ const message = {
         message,
         type
       })
+    },
+    $_showRegistrationError (message, response) {
+      let responseText = response ? response.data : ''
+      if (responseText.search('no free btc address to register') !== -1) {
```

Comment:
I guess for better readability we can use `.includes()` here, because it's doesn't require to compare with `-1`.

[String.prototype.includes()
](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/includes)

Reply:
As I understand you compare to error message that was sent from notary, is it okay? What notary team says about it? It is okay that we use their error exception to compare response and show notification to users? 🤔🤔🤔